jupyterhub/jupyterhub/base/jupyterhub-configmap.yaml

```diff
@@ -11,6 +11,7 @@ data:
   gpu_mode: ""
   singleuser_pvc_size: 2Gi
   notebook_destination: "$(notebook_destination)"
+  culler_timeout: "31536000" # 1 year in seconds
```

Comment:
Does this need to be a kfdef parameter so that user's can configure this from the kfdef `kustomizeConfig`?

Reply:
Since ODH JH is planned for deprecation and given the adiditonal work required to allow the user to customize this configmap, we can merge this as is for now.